docs/README.md

```diff
@@ -35,7 +35,7 @@ $ cd react/docs
 $ bundle exec rake
 $ bundle exec rake fetch_remotes
 $ bundle exec jekyll serve -w
-$ open http://localhost:4000/react/
+$ open http://localhost:4000/react/index.html
```

Comment:
Works for me…

![image](https://cloud.githubusercontent.com/assets/6820/19054469/60ea6122-8974-11e6-929c-9598a460c704.png)